### DISCUSSION TITLE
🚨 Deta Space and WebCrate Sunset on October 17, 2024

### DISCUSSION THREAD

**Discussion by BetaHuhn** [WebCrateApp/webcrate#90] [Announcements]
Dear WebCrate Community,

Deta, which [hosts and powers WebCrate](https://webcrate.app/docs/faq#whats-deta-space), has made the difficult decision to shut down [Deta Space](https://deta.space/sunset). You can continue to [login](https://deta.space/login) and use WebCrate for **45 days** until sunset on **October 17, 2024**, at which point Deta will start deleting all apps and data (_including WebCrate_).

Deta has built tools for exporting your data (and code for developers), with tutorials available in the [Deta Docs](https://deta.space/docs/en/sunset). To learn more about the Deta Space shutdown checkout [deta.space/sunset](https://deta.space/sunset). 

Additionally I am planning to finally add support for self hosting WebCrate outside of Deta Space and will provide easy import tools to get your data from Deta Space into a self hosted instance of WebCrate. This won't be a perfect replacement and will likely come with manual setup and work for you as a user, but will be the only way to continue using WebCrate after October 17, 2024.

The Deta team and me personally want to thank you for joining the journey so far. Everything we've done wouldn't have been possible without our community, investors, and friends.

To learn more about what Deta is doing next checkout [Deta Surf](https://deta.surf). We will onboard people in the very near future and it will be a good replacement for WebCrate.

See you around;
BetaHuhn

**Comment by vdbhb59** [comment]
> Additionally I am planning to finally add support for self hosting WebCrate outside of Deta Space and will provide easy import tools to get your data from Deta Space into a self hosted instance of WebCrate. This won't be a perfect replacement and will likely come with manual setup and work for you as a user, but will be the only way to continue using WebCrate after October 17, 2024.

FYI, please do not just have Docker as self-host option. Do allow for manual/via git, for easy setup. That would be real great for people like myself. :)

**Comment by marianoju** [comment]
> Additionally I am planning to finally add support for self hosting WebCrate outside of Deta Space and will provide easy import tools to get your data from Deta Space into a self hosted instance of WebCrate. This won't be a perfect replacement and will likely come with manual setup and work for you as a user, but will be the only way to continue using WebCrate after October 17, 2024.

How is that support for self hosting WebCrate outside of Deta Space coming along?
